dev/artifacts/Arcs/ArcMetadata.schema

```diff
@@ -15,3 +15,4 @@ schema ArcMetadata
     Boolean profile
   optional
     Text blurb
+    URL additionalRecipe
```

Comment:
Same question regarding the name of this.

Reply:
Changed to `externalManifest` per comment above.